apisix/stream/router/ip_port.lua

```diff
@@ -71,6 +96,20 @@ do
             end
 
             local route = item.value
+
+            if not item.value.remote_addr_matcher and item.value.remote_addr then
+                local ip = create_matcher(item.value.remote_addr)
+                if ip then
```

Comment:
We can write a new checker in https://github.com/apache/apisix/blob/df0db95ce8f8a5a977481539b6af7e056d295680/apisix/stream/router/ip_port.lua#L185 to ensure the matcher can be always created?

We can use the methods in https://github.com/apache/apisix/blob/master/apisix/core/ip.lua to do this.

Reply:
We can use validate_cidr_or_ip to validate.

Reply:
I think it's unnecessary to use `validate_cidr_or_ip` here, we just need to check the `remote_addr` and `server_addr` is nil or not. the `ipmatcher` will help to validate the address when create a new match object by using `new` method